client/my-sites/marketplace/pages/submission-success/signup-success.tsx

```diff
@@ -24,17 +29,35 @@ const SignupSuccess = () => {
 			/>
 			<MasterbarLogo />
 			<div className="signup-success">
-				<div className="signup-success__header">
-					<img src={ submissionSuccessImage } alt="Submission success" />
-					<h1 className="signup-success__header-title wp-brand-font">
-						{ translate( 'We’ll be in touch' ) }
-					</h1>
-					<p>
-						{ translate(
-							'Thank you for apply to join the WordPress.com marketplace. We’ll be in touch with you very soon to discuss next steps.'
-						) }
-					</p>
-				</div>
+				{ currentQuery?.returning !== 'true' ? (
+					<div className="signup-success__header">
+						<img src={ submissionSuccessImage } alt="Submission success" />
```

Comment:
This alt description doesn't seem very descriptive. Is it for the illustration? Should be left empty if so.

Reply:
I [created an issue](https://github.com/Automattic/wp-calypso/issues/66275) for this one